paperspace/cli/jobs.py

```diff
@@ -63,6 +64,35 @@ def list_jobs(api_key, **filters):
     command.execute(filters)
 
 
+@jobs_group.command("create", help="Create job")
+@click.option("--name", "name", help="Job name", required=True)
+@click.option('--machineType', 'machineType', help="Virtual machine type")
+@click.option('--container', 'container', help="Docker container")
+@click.option('--command', 'command', help="Job command/entrypoint")
+@click.option('--ports', 'ports', help="Mapped ports")
+@click.option('--isPublic', 'isPublic', help="Flag: is job public")
+@click.option("--workspace", "workspace", required=False, help="Path to workspace directory")
+@click.option("--workspaceArchive", "workspaceArchive", required=False, help="Path to workspace archive")
+@click.option("--workspaceUrl", "workspaceUrl", required=False, help="Project git repository url")
+@click.option("--workingDirectory", "workingDirectory", help="Working directory for the experiment", )
+@click.option('--experimentId', 'experimentId', help="Experiment Id")
+# @click.option('--envVars', 'envVars', help="Environmental variables ")  # TODO
+@click.option('--useDockerfile', 'useDockerfile', help="Flag: using Dockerfile")
+@click.option('--isPreemptible', 'isPreemptible', help="Flag: isPreemptible")
+@click.option('--project', 'project', help="Project name")
+@click.option('--projectHandle', '--projectId', 'projectHandle', help="Project handle", required=True)
+@click.option('--startedByUserId', 'startedByUserId', help="User ID")
+@click.option('--relDockerfilePath', 'relDockerfilePath', help="Relative path to Dockerfile")
+@click.option('--registryUsername', 'registryUsername', help="Docker registry username")
+@click.option('--registryPassword', 'registryPassword', help="Docker registry password")
+@common.api_key_option
```

Comment:
Make up your mind - it's either ' or " ;)
I suggest " as it is used in most of the existing code